docs/tutorials/quick-start/index.rst

```diff
@@ -0,0 +1,645 @@
+    Spec:
+      Acme:
+        Config:
+          Domains:
+            example.your-domain.com
+          Http 01:
+            Ingress:
+            Ingress Class:  nginx
+      Dns Names:
+        example.your-domain.com
+      Issuer Ref:
+        Kind:       Issuer
+        Name:       letsencrypt-staging
+      Secret Name:  quickstart-example-tls
+    Status:
+      Acme:
+        Order:
+          URL:  https://acme-staging-v02.api.letsencrypt.org/acme/order/7374163/13665676
+      Conditions:
+        Last Transition Time:  2018-11-17T18:05:57Z
+        Message:               Certificate issued successfully
+        Reason:                CertIssued
+        Status:                True
+        Type:                  Ready
+        Last Transition Time:  <nil>
+        Message:               Order validated
+        Reason:                OrderValidated
+        Status:                False
+        Type:                  ValidateFailed
+    Events:
+      Type     Reason          Age                From          Message
+      ----     ------          ----               ----          -------
+      Normal   CreateOrder     9m                 cert-manager  Created new ACME order, attempting validation...
+      Normal   DomainVerified  8m                 cert-manager  Domain "example.your-domain.com" verified with "http-01" validation
+      Normal   IssueCert       8m                 cert-manager  Issuing certificate...
+      Normal   CertObtained    7m                 cert-manager  Obtained certificate from ACME server
+      Normal   CertIssued      7m                 cert-manager  Certificate issued Successfully
+
+The events associated with this resource and listed at the bottom of the `describe` 
+results show the state of the request. In the above example the certificate was validated
+and issued within a couple of minutes.
+
+Once complete, cert-manager will have created a secret with the details of the 
+certificate based on the secret used in the ingress resource. You can use the 
+describe command as well to see some details:
+
+.. code-block:: shell
+
+    $ kubectl describe secret quickstart-example-tls
+
+    Name:         quickstart-example-tls
+    Namespace:    default
+    Labels:       certmanager.k8s.io/certificate-name=quickstart-example-tls
+    Annotations:  certmanager.k8s.io/alt-names=example.your-domain.com
+                  certmanager.k8s.io/common-name=example.your-domain.com
+                  certmanager.k8s.io/issuer-kind=Issuer
+                  certmanager.k8s.io/issuer-name=letsencrypt-staging
+
+    Type:  kubernetes.io/tls
+
+    Data
+    ====
+    tls.crt:  3566 bytes
+    tls.key:  1675 bytes
+
+
+Now that we have confidence that everything is configured correctly, you can update
+the annotations in the ingress to specify the production issuer:
+
+.. literalinclude:: example/ingress-tls-final.yaml
+   :language: yaml
+
+.. code-block:: shell
+
+   $ kubectl apply -f ingress.yaml
```

Comment:
They'll also need to manually delete the target secret at this point too, as we don't automatically detect the change in issuer and trigger a re-issue 😄

Reply:
Perfect, I'll add that into my sequence here!